client/views/admin/users/UserForm.js

```diff
@@ -66,7 +66,14 @@ export default function UserForm({ formValues, formHandlers, availableRoles, app
 						<Field>
 							<Field.Label>{t('Name')}</Field.Label>
 							<Field.Row>
-								<TextInput error={errors && errors.name} flexGrow={1} value={name} onChange={handleName} />
+								<TextInput
+									error={errors && errors.name}
+									type='text'
+									pattern='[A-Za-z ]+[ A-Za-z]+'
```

Comment:
Why? What's the reasoning behind this? Why should we not allow a name like "Clarinet 4 my boyfriend"? 🤔 

Remember, Rocket.Chat can be used by anyone, closed family, corporate or public communities where users may not want to use their original name for whatever reason.

PS: this doesn't cater to the registration form.